rpc/rpc-wrappers.go

```diff
@@ -1201,13 +1201,7 @@ func NewStorageAuthorityServer(rpc Server, impl core.StorageAuthority) error {
 		}
 
 		err = impl.AddSCTReceipt(core.SignedCertificateTimestamp(sct))
-		if err != nil {
-			// AUDIT[ Error Conditions ] 9cc4d537-8534-4970-8665-4b382abe82f3
-			errorCondition(MethodAddSCTReceipt, err, req)
```

Comment:
If you look at the surrounding functions, we don't call `errorCondition` on ordinary error returns, only when we fail to unmarshal the JSON. Errors returned from the underlying function are passed along to the caller to handle.